src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -1163,6 +1169,20 @@ private void AssertSnmp(ServerStore serverStore, LicenseStatus licenseStatus)
             throw new LicenseLimitException(LimitType.Snmp, message);
     }
 
+    private void AssertSchemaValidationConfiguration(DatabaseRecord databaseRecord, LicenseStatus licenseStatus, ClusterOperationContext context)
+    {
+        if (licenseStatus.HasSchemaValidation)
+            return;
+
+        if (databaseRecord.SchemaValidation == null || databaseRecord.SchemaValidation.Disabled)
+            return;
+
+        if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion72000) == false)
```

Comment:
This isn't needed.
We add this only when we add a license limitation to an existing feature after it was released.